packages/runtime/datastore/src/localChannelStorageService.ts

```diff
@@ -13,13 +13,29 @@ export class LocalChannelStorageService implements IChannelStorageService {
     }
 
     public async read(path: string): Promise<string> {
-        const contents = this.readSync(path);
-        return contents !== undefined ? Promise.resolve(contents) : Promise.reject(new Error("Not found"));
+        const blob = await this.readBlob(path);
+        assert(blob !== undefined, "Not Found");
+        assert(blob.contents !== undefined, "Not Found");
```

Comment:
Message on this assert is not entirely correct, and I'd say it's not such an important assert - compiler already does good job ensuring that contents is not undefined (we usually do not check such conditions in other parts of the code)